src/components/pages/account/TransactionsTable.tsx

```diff
@@ -118,9 +118,7 @@ function useTransaction(guid: string): UseQueryResult<Transaction> {
       () => Transaction.findOne({
         where: { guid },
         relations: {
-          splits: {
-            fk_account: true,
-          },
+          splits: true,
```

Comment:
We load the accounts using `useAccounts` instead of querying with the transaction (so path is calculated)